src/commands/mapsheet-coverage/__test__/mapsheet.coverage.test.ts

```diff
@@ -112,6 +122,10 @@ describe('mapsheet-coverage', () => {
       id: 'layer-a-id',
       license: 'CC-BY-4.0',
       source: 'ms://layers/a/collection.json',
+      host: '',
```

Comment:
I would prefer we don't set things to "" if it doesn't exist it should just be undefined.

Reply:
I like including it, since it means users will know that it's empty, rather than not knowing that it exists. But I'd absolutely leave this up to you guys. @dwsilk?

Reply:
Let's just leave it out completely, I'm curious to see what the end result will look like in GIS software.